dockerfiles/run/guice/jpa-smtp/destination/run_james.sh

```diff
@@ -2,4 +2,7 @@
 if [ "$GLOWROOT_ACTIVATED" == "true" ]; then
     GLOWROOT_OPTIONS=-javaagent:/root/glowroot/glowroot.jar
 fi
-java -classpath '/root/james-server-jpa-smtp-guice.jar:/root/james-server-jpa-smtp-guice.lib/*' -javaagent:/root/james-server-jpa-smtp-guice.lib/openjpa-3.1.0.jar $GLOWROOT_OPTIONS -Dlogback.configurationFile=/root/conf/logback.xml -Dworking.directory=/root/ $JVM_OPTIONS org.apache.james.JPAJamesServerMain
\ No newline at end of file
+java -Dlogback.configurationFile=/root/conf/logback.xml \
```

Comment:
the command before was a bit different with more arguments... Is it just something outdated or something got forgotten here?

Reply:
OUPS